TRANSPORT Macro (#83)
syslog-ng/syslog-ng.github.io: doc/_admin-guide/110_Template_and_rewrite/000_Customize_message_format/004_Macros_of_syslog-ng.md

An alias of the ${TZOFFSET} macro.

## ${TRANSPORT}

> This macro is available in syslog-ng OSE 4.5 and later versions. The syslog-ng OSE application automatically populates this name-value pair with the transport-mechanism used to retrieve the message. The value depends on the source driver that received the message. The implemented values are the following:

Reviewer: ```suggestion
This macro is available in {{ site.product.short_name }} 4.5 and later versions. The {{ site.product.short_name }} application automatically populates this name-value pair with the transport-mechanism used to retrieve the message. The value depends on the source driver that received the message. The implemented values are the following:
```

> * BSD syslog drivers `tcp()`, `udp()` & `network()`

Reviewer: ```suggestion
* BSD syslog drivers [[tcp(), udp()|adm-src-tcp]] & [[network()|adm-src-netw]]
```

>   * RFC-3164 + tls

Reviewer 2: the exact string looks like `rfc3164+tls`

>   * RFC-3164 + tcp

Reviewer 2: the exact string looks like `rfc3164+tcp`

>   * RFC-3164 + udp

Reviewer 2: the exact string looks like `rfc3164+udp`

>   * RFC-3164 + proxied-tls

Reviewer 2: the exact string looks like `rfc3164+proxied-tls`

>   * `rfc3164+<custom logproto like altp>`

Reviewer: \<\> needs to be escaped

```suggestion
  * `rfc3164+\<custom logproto like altp\>`
```

>   * RFC-3164 + \<custom logproto like altp\>

Reviewer 2: the exact string looks like `rfc3164+<custom logproto like altp>`

> 

Reviewer 2: Missing cases and strings are:

`rfc3164+proxied-tcp`
`rfc3164+proxied-tls-passthrough`

* UNIX domain drivers `unix-dgram()`, `unix-stream()`
  * `unix-stream`
  * `unix-dgram`

* RFC5424-style syslog `syslog()`:

>   * `rfc5426`: syslog over udp

Reviewer: ```suggestion
  * RFC-5426: syslog over udp
```
this will autolink to the RFC

>   * RFC-5426: syslog over udp

Reviewer 2: the exact string looks like `rfc5426`

>   * RFC-5425: syslog over tls

Reviewer 2: the exact string looks like `rfc5425`

>   * RFC-6587: syslog over tcp

Reviewer 2: the exact string looks like `rfc6587`

>   * RFC-5424 + \<custom logproto like altp\>: syslog over a logproto plugin

Reviewer 2: the exact string looks like `rfc5424+<custom logproto like altp>`

> 

Reviewer 2: Missing cases and strings are

`rfc6587+proxied-tcp`
`rfc5424+proxied-tls`
`rfc5424+proxied-tls-passthrough"`

* Other drivers:
  * otlp: `otel()` driver
  * mqtt: `mqtt()` driver

>   * hypr-api: `hypr-audit-source()` driver

Reviewer: Cannot find a doc for this...

This macro is available in syslog-ng OSE 4.5 and later versions. The syslog-ng OSE application automatically populates this name-value pair with the transport-mechanism used to retrieve the message. The value depends on the source driver that received the message. The implemented values are the following:

* BSD syslog drivers `tcp()`, `udp()` & `network()`
  * rfc3164+tls
  * rfc3164+tcp
  * rfc3164+udp

>   * rfc3164+proxied-tls

Reviewer 2: these 

`rfc3164+proxied-tcp`
`rfc3164+proxied-tls-passthrough`

are still missing

  * rfc3164+ \<custom logproto like altp\>

* UNIX domain drivers `unix-dgram()`, `unix-stream()`
  * `unix-stream`
  * `unix-dgram`

* rfc5424 style syslog `syslog()`:
  * rfc5426+udp
  * rfc5425+tls
  * rfc6587+tcp

>   * rfc5424+\<custom logproto like altp\>: syslog over a logproto plugin

Reviewer 2: i'd move this to the last position

  * rfc5424+proxied-tls-passthrough

> 

Reviewer 2: these 

`rfc6587+proxied-tcp`
`rfc5424+proxied-tls`

are still missing

  * rfc3164+tls
  * rfc3164+tcp
  * rfc3164+udp
  * rfc3164+proxied-tcp
  * rfc3164+proxied-tls
  * rfc3164+proxied-tls-passthrough
  * rfc3164+ \<custom logproto like altp\>

* UNIX domain drivers `unix-dgram()`, `unix-stream()`
  * `unix-stream`
  * `unix-dgram`

* rfc5424 style syslog `syslog()`:

>   * rfc5424+proxied-tls

Reviewer 2: just curious, why not before the 

`rfc5424+proxied-tls-passthrough`

like in the previous case

Author: No particular reason, but you're right, it makes more sense there. Updated.
